removal-constants.js

```diff
@@ -53,13 +52,14 @@ const REMOVAL_CONSTANTS = {
     {
       id: 1,
       name: "group_01",
-      start_time: 1641916800, //wed, jan 11 8am PST
+      start_time: 1642197405, //MH TODO: remove after testing
```

Comment:
Does this need to be removed?

Reply:
I kept it in so QA could test - will change now for launch.